tests/unit/certGen.unitTest.ts

```diff
@@ -3901,7 +3901,234 @@ describe("cert-gen", () => {
                 )
             );
             const testResult: any = JSON.parse(event.Records[0].body);
+            const hgvWithDefectRectifiedAtTest: any = JSON.parse(event.Records[3].body);
```

Comment:
Can you move this into the context below

Reply:
moved